applications/surgical_scene_recon/run_complete_workflow.py

```
@@ -0,0 +1,403 @@
+        cmd.append("--no-loop")
+    else:
+        # num_frames == -1: render all frames in continuous loop (default)
+        cmd.append("--loop")
+
+    print(f"Command: {' '.join(cmd)}\n")
+    print("Starting visualization...\n")
+
+    try:
+        if visualize:
+            # Run with display
+            ret = subprocess.call(cmd)
```

Ruff (0.14.5)

`subprocess` call: check for execution of untrusted input

(S603)